lib/puppet/provider/rbac_api.rb

```diff
@@ -16,7 +16,13 @@ class Puppet::Provider::Rbac_api < Puppet::Provider
   def self.build_auth(uri)
     https = Net::HTTP.new(uri.host, uri.port)
     https.use_ssl = true
-    https.ssl_version = :TLSv1
+
+    if Puppet.version.split('.')[0] = '6'
```

Comment:
What do you think about using `Puppet::Util::Package.versioncmp(Puppet.version, '6.0')` instead to make it more future-proofed?

Reply:
That's a much better solution. Let me get that change put in.